### DISCUSSION TITLE
当前 mihomo v1.18.10 (2024/11/8) 是否支持原版 clash 配置文件？

### DISCUSSION THREAD

**Discussion by m2acgi** [MetaCubeX/mihomo#1644] [Q&A]
当前 mihomo v1.18.10 (2024/11/8) 是否支持原版 clash 配置文件？

**Comment by AtticusZeller** [comment]
Not 100%,vless can be handled via meta but the classical clash premium core must failed

**Comment by wyf9** [comment]
支持
